### PR TITLE
Add Russian localization for CLI

### PR DESCRIPTION
**Pull request by lifefire1**
I added a file with the Russian localization for cli

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8132"></a>

### CODE REVIEW

**Comment by lifefire1** [comment]
and another question, I can't find the tasks that need to be done, could you tell me where to find them?

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties
Comment (MarkEWaite) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:26]: Optional, would be nice to not have the trailing space character at the end of the line.
```suggestion
CLI.BadAuth=Параметры окружения JENKINS_USER_ID и JENKINS_API_TOKEN должны быть оба установлены или оставлены пустыми.
```

**Comment by MarkEWaite** [comment]
> and another question, I can't find the tasks that need to be done, could you tell me where to find them?

I'm not sure if these pages will answer your question about tasks that need to be done, but we recently published the following pages related to internationalization and localization.

* [Internationalization and localization](https://www.jenkins.io/doc/developer/internationalization/) including [translator tasks](https://www.jenkins.io/doc/developer/internationalization/#translator-tasks)
    * [Generate property files and translate messages in properties files](https://www.jenkins.io/doc/developer/internationalization/translate-a-properties-file/)
    * [Translate help files](https://www.jenkins.io/doc/developer/internationalization/translate-a-help-file/)
    * [Submit a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

Are they enough to answer the question or is more information needed?

**Review by MarkEWaite — APPROVED** [review]
I passed many of the strings to translate.google.com and the translation back to English seemed reasonable to me.  One optional comment is included in the review, but I approve the pull request whether that comment is adopted or not.

**Comment by lifefire1** [comment]
I just wanted to improve my programming skills and see the tasks that need to be implemented. It seems that there is a jira for this, I just can't find a link to it.

**Comment by MarkEWaite** [comment]
> It seems that there is a jira for this, I just can't find a link to it.

I don't see an open [Jira ticket](https://issues.jenkins.io/issues/?jql=text%20~%20Russian%20and%20status%20not%20in%20(closed%2C%20resolved)%20ORDER%20BY%20key%20ASC) specific to Russian localization of command line interface.  There are a few open tickets related to Russian translation, but nothing that seems specific to this pull request.

**Review by lifefire1 — COMMENTED** [review]
the last space has been removed

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
